.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
We can also explain the --assume-unchanged git flag, that would allow to stop tracking changes to a tracked files.